graphql/Cargo.toml

```diff
@@ -1,15 +1,17 @@
 [package]
 name = "graphql-rs"
 version = "0.1.0"
+description = "An implementation of GraphQL for Rust."
+keywords = ["graphql", "core"]
 authors = ["cscks <cs.cksac@gmail.com>"]
 license = "MIT"
 readme = "README.md"
 repository = "https://github.com/cksac/graphql-rs"
 homepage = "https://github.com/cksac/graphql-rs"
-description = """
-A implementation of GraphQL for Rust
-"""
-keywords = ["graphql"]
+
+[lib]
+name = "graphql"
```

Comment:
Does this mean we declare dep `graphql-rs = "*"` but import use `use graphql`?
I would keep it as "graphql_rs" when import, as user don't need to remember such exception.
User can do `extern crate graphql_rs as graphql;` if wanted.